sup3r/bias/bias_calc.py

```diff
@@ -581,12 +598,30 @@ def fill_extend(self, out, smooth_extend):
         for key, arr in out.items():
             nan_mask = np.isnan(arr[..., 0])
             for idt in range(self.NT):
-                arr[..., idt] = nn_fill_array(arr[..., idt])
+
+                arr_smooth = arr[..., idt]
+
+                needs_fill = (np.isnan(arr_smooth).any() and fill_extend
+                              or smooth_interior > 0)
```

Comment:
You should be cautious with these kinds of operations, logical operators have an order of operations priority that is not intuitive. It's fine as-is but I would use parenthesis to make the order of operations explicit.

Reply:
Yeah good call, it's bitten me in the past.